bin/docker-entrypoint.sh

```diff
@@ -18,11 +18,9 @@ if [[ $LOCALSTACK_API_KEY ]] && ! compgen -G /usr/lib/localstack/.*pro-version >
     echo ""
 fi
 
-# Strip `LOCALSTACK_` prefix in environment variables name (except
-# LOCALSTACK_HOST and LOCALSTACK_HOSTNAME)
+# Strip `LOCALSTACK_` prefix in environment variables name (except LOCALSTACK_HOST)
 source <(
   env |
-  grep -v -e '^LOCALSTACK_HOSTNAME' |
```

Comment:
or should we keep it here to get deprecation warnings in the container?

Reply:
re-added

Reply:
`LOCALSTACK_HOSTNAME` is not deprecated, it's ignored...

... but the message in `deprecations.py` is wrong. Could you update the messages for the deprecation variables to say "This option has no effect. Please use LOCALSTACK_HOST instead"? After that, then I think we should keep this line so the user gets deprecation warnings, though CLI users will get deprecation warnings twice. Doubly informed I say!